metadata/metadata.json

```diff
@@ -0,0 +1,47 @@
+   "matcherProtection" : 1,
+   "supportedExtensions" : [
+      {
+         "id" : "hmac-secret",
+         "fail_if_unknown" : false
+      },
+      {
+         "id" : "credProtect",
+         "fail_if_unknown" : false
+      }
+   ],
+   "cryptoStrength" : 128,
+   "description" : "OpenSK authenticator",
+   "authenticatorVersion" : 1,
+   "isSecondFactorOnly" : false,
+   "userVerificationDetails" : [
+      [
+         {
+            "userVerification" : 1
+         },
+         {
+         	"userVerification" : 4
```

Comment:
nit: indentation differs

Reply:
Nice catch. Tab vs. spaces. That's the drawback of having edited this file with different editors :)